mappings/net/minecraft/server/dedicated/ServerMBean.mapping

```diff
@@ -0,0 +1,45 @@
+CLASS net/minecraft/class_5169 net/minecraft/server/dedicated/ServerMBean
```

Comment:
Humm, im not too sure about this name.

Reply:
This is like a class that implements `DynamicMBean`, which the dynamic mbean supports retrieving information on server avg tick time and past tick times.